monitor/services/src/main/java/org/datacleaner/monitor/configuration/TenantContextFactory.java

```diff
@@ -29,4 +32,8 @@
     public TenantContext getContext(TenantIdentifier tenant);
     
     public TenantContext getContext(String tenantId);
+
+    public Iterator<RepositoryFolder> getRepositoryFolderIterator();
+
+    public Iterator<TenantContext> getActiveTenantContextIterator();
```

Comment:
Could we instead return Iterable here, and not add the return-type class to the method name, so it will just be "getActiveTenantContexts()" ?